### PR TITLE
[CI] Pull request labeler

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9990f96fd4495fb0aa19ed4cac7bce4fede8c51f at: https://63898db37c3ff633326f4238--layer5.netlify.app

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+area/docs:
+  - "readme.md"
+  - "contributing.md"
+area/ci:
+  - ".github/**/*"
+  - ".husky/**/*"
+area/blog:
+  - "src/collections/blog/**/*"
+  - "src/pages/blog/**/*"
```
Comment (Nikhil-Ladha) [inline .github/labeler.yml:9]: Curious if something like: `src/templates/blog-*` works? 🤔

Comment (Nikhil-Ladha) [reply .github/labeler.yml:9]: If yes, I can push a couple more paths to the PR

Comment (Nikhil-Ladha) [reply .github/labeler.yml:9]: Seems like it will work.

**Comment by Nikhil-Ladha** [comment]
@leecalcote added a couple more.
If everything looks in order to you, feel free to merge ✅

**Comment by l5io** [comment]
🚀 Preview for commit 1b1a35c701fb6d26c35b18b12c5c7e45e4e6fac8 at: https://6389b47030b5194eb13da905--layer5.netlify.app

**Review by leecalcote — COMMENTED** [review]
Thank you, sir.

**Pull request merged**